packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -57,9 +57,25 @@ const Web3 = require("web3");
 const { delay, createNewLogger } = require("@uma/financial-templates-lib");
 let customLogger;
 let spokeUrl;
+let spokeUrlTable = {};
 let customNodeUrl;
 let hubConfig = {};
 
+// Lets us specify spoke url by a label or fallback to default spoke pool url.
+// this should allow us to create multiple levels of spoke pool hardware (small,medium,large)
+// and switch between urls based on the bot config.
+function getSpokeUrl(label) {
+  if (label) {
+    // this will check if you have specified a label, and do a lookup. If a label is specified but does not exist this will be an error
+    const url = spokeUrlTable ? spokeUrlTable[label] : undefined;
```

Comment:
Would this do the same thing?
```suggestion
    const url = spokeUrlTable?.[label];
```

Reply:
yea good call